EosGround/database/models/eos/terminal_output.py

```diff
@@ -0,0 +1,14 @@
+from sqlalchemy import Identity, Integer, ForeignKey, TEXT
+from sqlalchemy.orm import Mapped, mapped_column
+
+from EosGround.database.models import TableBase
+from EosGround.database.models.eos import SCHEMA
+
+
+class Telemetry(TableBase):
```

Comment:
Is the class suppose to be named Telemtry? Shouldn't it be named TerminalOutput?

Reply:
that was a great catch, thank you!